db/schema.rb

```diff
@@ -468,6 +467,31 @@
   add_foreign_key "patients", "addresses"
   add_foreign_key "protocol_drugs", "protocols"
 
+  create_view "blood_pressures_per_facility_per_days", materialized: true, sql_definition: <<-SQL
```

Comment:
Double-checked. All the view stuff is reordering. I believe Rob also found that this is a known issue with `scenic`